Patch by schwardt@univention.de (Thu Sep 10 10:53:55 2009 +0200):

univention: changed objectclasses to UCS standard



--- a/horde-webmailer/lib/Horde/Kolab/Server/Object.php
+++ b/horde-webmailer/lib/Horde/Kolab/Server/Object.php
@@ -35,7 +35,7 @@ define('KOLAB_ATTR_GIVENNAME',    'givenName');
 define('KOLAB_ATTR_FN',           'fn');
 define('KOLAB_ATTR_LNFN',         'lnfn');
 define('KOLAB_ATTR_FNLN',         'fnln');
-define('KOLAB_ATTR_MAIL',         'mail');
+define('KOLAB_ATTR_MAIL',         'mailPrimaryAddress');
 define('KOLAB_ATTR_ALIAS',        'alias');
 define('KOLAB_ATTR_SID',          'uid');
 define('KOLAB_ATTR_ACL',          'acl');
@@ -60,7 +60,7 @@ define('KOLAB_OC_TOP',                'top');
 define('KOLAB_OC_INETORGPERSON',      'inetOrgPerson');
 define('KOLAB_OC_KOLABINETORGPERSON', 'kolabInetOrgPerson');
 define('KOLAB_OC_HORDEPERSON',        'hordePerson');
-define('KOLAB_OC_KOLABGROUPOFNAMES',  'kolabGroupOfNames');
+define('KOLAB_OC_KOLABGROUPOFNAMES',  'univentionGroup');
 define('KOLAB_OC_KOLABSHAREDFOLDER',  'kolabSharedFolder');
 
 /** Define the possible Kolab user types */
